genome_annotation_tools.yml

```diff
@@ -101,9 +101,9 @@ tools:
   owner: bgruening
   tool_panel_section_label: "Gene Prediction"
 
-- name: glimmer3
-  owner: bgruening
-  tool_panel_section_label: "Gene Prediction"
+#- name: glimmer3
```

Comment:
This one should have conda dependencies now and are hosted under IUC. (the owner is still bgruening)